training_day1/src/components/inpText.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import Input from "./Input";
+
+function InputText(props) {
+    const [value, setValue] = useState(props.value);
+    function handleChange(e) {
+        console.log("clicked");
```

Comment:
remove the console and while setting value, remove the whitespace from both sides of a string before setting the value